changelogs/unreleased/gh-8605-add-gethostname.md

```diff
@@ -0,0 +1,3 @@
+## feature/box
+
+* Added hostname to box.info, it is obtained on request (gh-8605).
```

Comment:
```suggestion
* Added a new `box.info` parameter `hostname` (gh-8605).
```

Though I don't know if "It is obtained on request" is really important for a brief changelog entry. You can leave it if it is. Just make is a full separate sentence.

Reply:
Agreed.